ibis/backends/bigquery/__init__.py

```diff
@@ -683,12 +683,31 @@ def _to_sqlglot(
         self._define_udf_translation_rules(expr)
         sql = super()._to_sqlglot(expr, limit=limit, params=params, **kwargs)
 
+        table_expr = expr.as_table()
+        geocols = [
+            name for name, typ in table_expr.schema().items() if typ.is_geospatial()
+        ]
+
         query = sql.transform(
             _qualify_memtable,
             dataset=getattr(self._session_dataset, "dataset_id", None),
             project=getattr(self._session_dataset, "project", None),
         ).transform(_remove_null_ordering_from_unsupported_window)
-        return query
+
+        if not geocols:
+            return query
+
+        quoted = self.compiler.quoted
+        return sg.select(
+            sge.Star(
+                replace=[
```

Comment:
I'm not sure I quite follow what is happening here. Are we converting text into binary for geo columns, or what is it exactly that we are returning instead of `query` here? Can you explain it in a comment, or in the main comment in the PR what the problem was and how this fixes it?

I can't run the bigquery backend on my laptop so a bit more context would be helpful

Reply:
I'll add a comment, but this is the same thing we had to do in duckdb to get it to properly display geometries in interactive mode, though here it's for a slightly different reason.

Reply:
Added a comment in the latest commit.